### PR TITLE
Closes issue #771 - Remove unused strings

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson @thisandagain what is the process for review here?

Note please use this link to review https://github.com/mozilla/webmaker-app/pull/786/files?w=1 since I have change the spacing in this file.

File: locale/en_US/mobile-appmaker.json [hunk 1]
```diff
@@ -1,438 +1,302 @@
...
+  },
+  "Add a map, a emergency call button, and other tips to keep safe in your community": {
+    "message": "Add a map, a emergency call button, and other tips to keep safe in your community",
+    "description": "description for template"
+  },
+  "Text Box": {
+    "message": "Text Box",
+    "description": "Type of block"
+  }
+}
```
Comment (k88hudson) [inline locale/en_US/mobile-appmaker.json]: add newline

**Comment by k88hudson** [comment]
Sorry for the delay. One small nit, just add back the newline on the JSON file

**Comment by k88hudson** [comment]
after that go ahead and merge

**Pull request merged**